documentation/modules/security/proc-renewing-your-own-ca-certificates.adoc

```diff
@@ -54,6 +54,11 @@ For example `ca-2018-09-27T17-32-00Z.crt`.
 +
 [source,shell,subs="+quotes"]
 mv ca.crt ca-$(date -u -d"$(openssl x509 -enddate -noout -in ca.crt | sed 's/.*=//')" +'%Y-%m-%dT%H-%M-%SZ').crt
++
+Saving the current CA certificate has to be done only if you are using a different key for generating the new certificate.
+In this case the CN (Common Name) of the new CA certificate should be different from the previous one.
+For example, when Strimzi cluster operator takes care of the renewal automatically it adds a suffix like "vX" to identify a version "X" (that is, v0, v1, and so on).
+If you are using the same key for generating the new CA certificate saving the current one is not necessary because only one CA certificate at time can be trusted.
```

Comment:
IIRC, you said that saving the old here caused failures in the scripts. So maybe this should use stronger wording?

Reply:
Yeah openssl is not able to deal with two CA having same CN. I will be clearer about this.